go.mod

```diff
@@ -27,6 +27,7 @@ require (
 	github.com/openshift/machine-config-operator v4.2.0-alpha.0.0.20190917115525-033375cbe820+incompatible
 	github.com/openshift/ptp-operator v0.0.0-20200421195159-92f3d20c48f9
 	github.com/openshift/sriov-network-operator v0.0.0-20200428080346-d046d9acc3dd
+	github.com/openshift/sriov-tests v0.0.0-20200430133953-189e0fb45a7b
```

Comment:
This is not good we should not use the sriov-tests any more.

sriov-tests is only for the 4.4 branch